### DISCUSSION TITLE
Modifying the main menu

### DISCUSSION THREAD

**Discussion by shoaib-jamal** [prochazkaml/MichalOS#7] [Ideas]
A Reboot or Restart option beneath the Power Off  menu would look good. Maybe you could code a restart.asm option just like you did in shutdown.asm

**Comment by prochazkaml** [comment]
Yeah, that would probably be better, wouldn't it? 😁

**Reply by shoaib-jamal** [reply]
Yeah👍

**Reply by shoaib-jamal** [reply]
Hey did you get your website domain (.ml) from freenom?

**Reply by prochazkaml** [reply]
Yes, and I can see that you have as well with .tk :)

**Reply by shoaib-jamal** [reply]
If you're not on Social media then maybe on email? Because there are some private talks here so mail would be fine for some private discussions

**Reply by prochazkaml** [reply]
I've added my e-mail address to my GitHub profile, so you can contact me there.

**Reply by shoaib-jamal** [reply]
Ok
